blueocean-core-js/src/js/config.js

```diff
@@ -85,6 +87,13 @@ export default {
         return (typeof config.jenkinsRootURL === 'string' ? config.jenkinsRootURL : '/jenkins');
     },
 
+    getResourceURL() {
+        if (!config.isLoaded) {
+            this.loadUrls();
+        }
+        return (typeof config.resourceUrl === 'string' ? config.resourceUrl : '/jenkins/static');
```

Comment:
@michaelneale the above is a replicate from e.g. `getJenkinsRootURL` https://github.com/jenkinsci/blueocean-plugin/blob/master/blueocean-core-js/src/js/config.js#L85 I guess that it was the solution to the weird bug ;)